cluster/images/provider-terraform/terraformrc.hcl

```diff
@@ -0,0 +1 @@
+
```

Comment:
why do we need this file?

Reply:
This file is part of the upjet Dockerfile and contained some plugin dir cache statements. It is empty so I will remove it for now